nautobot/ipam/migrations/0020_aggregate_to_prefix_data_migration.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, time
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+from django.utils.timezone import make_aware
+
+from nautobot.ipam import choices
+
+
+def migrate_aggregate_to_prefix(apps, schema_editor):
+
+    Prefix = apps.get_model("ipam", "Prefix")
+    Aggregate = apps.get_model("ipam", "Aggregate")
+    Status = apps.get_model("extras", "Status")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    prefix_ct = ContentType.objects.get_for_model(Prefix)
+
+    try:
+        prefix_default_status = Status.objects.get(content_types=prefix_ct, slug="active")
+    except ObjectDoesNotExist:
+        prefix_default_status = Status.objects.filter(content_types=prefix_ct).first()
```

Comment:
Corner case, but should we explicitly check for the case where there are *no* prefix Statuses?